deliciousbrains/wp-post-promoter
# Update build dependencies and Gulpfile

I got a bunch of dependencies after merging #10.

This PR updates all of the build tooling and the Gulpfile.

## Testing

1. Pull the changes
2. Run each of the gulp tasks
3. Ensure that the CSS and JS output hasn't changed (except for variable names and whitespace)
